tests/scheduler_test.rs

```diff
@@ -0,0 +1,60 @@
+use std::boxed::Box;
+use std::cell::RefCell;
+use std::rc::Rc;
+use yew::scheduler::{scheduler, Runnable, Scheduler};
```

Comment:
We can add

```rust
#[cfg(feature = "wasm_test")]
use wasm_bindgen_test::wasm_bindgen_test as test;
```

to make sure this test runs with the `cargo test --features wasm_test --target wasm32-unknown-unknown` command